src/config.rs

```diff
@@ -398,14 +398,17 @@ pub enum UserDisplayStyle {
     DisplayName,
 }
 
-#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq)]
 #[serde(rename_all = "lowercase")]
 pub enum NotifyVia {
     /// Deliver notifications via terminal bell.
     Bell,
     /// Deliver notifications via desktop mechanism.
     #[cfg(feature = "desktop")]
     Desktop,
+
+    #[serde(untagged)]
+    Multiple(Vec<Self>),
```

Comment:
This technically allows `[["bell", "bell"], "desktop"]` and other silly settings, but for code simplicity that feels fine to me. The other approach would be to use something like `bitflags` but I'm not sure how complicated deserializing that would be.

Reply:
I think that making this look like a bitwise-or would be good so that folks can write `via: "bell|desktop"`. There's an example of what that looks like [here for VimModes](https://github.com/ulyssa/iamb/blob/3355eb2d2622f28d2fed6eb00a6aa78868a87b83/src/config.rs#L181-L223) which you could copy for making a `NotifyViaSet(Vec<NotifyVia>)` type.

Reply:
Finally got around to changing it to look like a bitwise-or, and added parser tests for it.